index.html

```diff
@@ -0,0 +1,67 @@
+    <a class="nav-link" href="#Recipe">Soyabean chhoila Recipe</a>
+    <a class="nav-link" href="#contact">Contact</a></nav>
+</header> 
+<main>
+        <h1>About</h1>
+    <section id="about">
+    <p> Soya chunks are made from defatted soy flour, a by-product of extracting soybean oil. It has a protein content which is equal to that of meat. It is quick and also easy to cook.Since I am a vegeterian , it is my one of the favourite foods to eat and cook.I want to dedicate this recipe to my vegetirain friends and for meat lovers too. This is an easy way to make soyabean Choila.</p>
+<img src=" https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJo3YuShJTRNGRhJtZL5XC3hKuHEyaZQc6wg&usqp=CAU" alt="soyabean chhoila phot0" width="300"></section>
+<h2>Soyabean chhoila Receipe:Ingredients</h2>
+<section id="recipe">
+<ul>Soy Chunks (1.5 lbs)</ul>
+ <ul>Onion (Medium sized, finely sliced)</ul> 
+<ul>Serrano Peppers (Green Chilies, 2-3)</ul>
+<ul>Ginger-Garlic Paste (1 table spoon)</ul>
+<ul>Ginger (2 table spoons, thinly sliced)</ul>
+<ul>Garlic (2 table spoons, thinly sliced)</ul>
+<ul>Salt (according to the taste)</ul>
+<ul>Turmeric (1 tea spoon)</ul>
+<ul>Red Chili Powder (1/2 table spoon)</ul>
+<ul>Schezwan Pepper (1 and 1/2 table spoons)</ul>
+<ul>Cumin-Coriander Powder (1 table spoon)</ul>
+<ul>omatoes (3 medium sized, boiled and peeled)</ul>
+<ul>Fenugreek seeds (1 tea spoon)</ul>
+<ul>Oil (2 table spoons)</ul>
+<ul>Lime juice (3 table spoons)/ul></ul></section>
+<h1>Directions:</h1>
+<ol>
+<li>Soak soy chunks on a cold water for 15 minutes.</li>
+<img src=" https://1.bp.blogspot.com/-vhIjSXOXcsk/UvF9ST8dXfI/AAAAAAAAA8w/k0YDKotbTuo/s1600/DSC03309.JPG" width="200">
+<li>Heat pan and add a tea spoon of oil and fry soya chunks until red and crispy.</li>
+<img src=" https://4.bp.blogspot.com/-5ghySqG2YAY/UvF9WHgCXoI/AAAAAAAAA84/QYpJ26rdhDQ/s1600/DSC03311.JPG" width="200">
+<li>On a large bowl, place soya chunks, add all of the above mentioned ingredients except fenugreek seeds, lime juice and remaining oil.</li>
+<img src="https://2.bp.blogspot.com/-sMPBM3oBqyU/UvF9ultY6xI/AAAAAAAAA9A/CZRa-4EqGHw/s1600/DSC03314.JPG" width="200">
+<li>Heat remaining oil and add fenugreek. Fry until brownish black and pour it over the mixture.</li>
+<img src=" https://2.bp.blogspot.com/-xmgI-deBIDo/UvF9EKkzU7I/AAAAAAAAA8o/Dp9AWDIutBE/s1600/DSC03313.JPG" width="200">
+<li>Add lime juice and give everything a good mix.</li>
+<img src="https://3.bp.blogspot.com/-EYljc9Iut4Y/UvGC_45KGMI/AAAAAAAAA9g/kpwmfb1cHPA/s1600/DSC03318.JPG" width="200" height="200"></ol>
+<p>Done! Simple as that. you can watch this video too</p>
+   <div> <iframe id="video" title="Simple way to make soyabean chhoia" width="300" height="300" src="https://www.youtube.com/embed/oGkWnSLPZYM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></div>
+
+ 
+    <h1>Contact</h1>
```

Comment:
typically you only put 1 `h1` tag on the page. Google won't put you high up in the search results otherwise. Try using a `h2`

Reply:
thank you